fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/metadata/IndexAggregateFunction.java

```diff
@@ -122,7 +125,15 @@ public int hashCode() {
     }
 
     @Override
-    public int planHash() {
-        return name.hashCode() + operand.planHash() + Objects.hashCode(index);
+    public int planHash(@Nonnull final PlanHashKind hashKind) {
```

Comment:
I think moving the `planHash()` (the one without parameters) to a default in the `PlanHashable` interface changes the code Java generates for callers if they use a specific reference to this class. This means that you probably create a binary incompatibility meaning that clients of the library (including CK) would have to be recompiled against the new RL library.

Reply:
I think this should be notes in the release notes of this PR.

Reply:
Done